File: csc106/Assignment 3/sort3.cpp
// Oliver Tonnesen
// V00885732

#include <iostream>
#include <bits/stdc++.h>
using namespace std;

int main(void) {
	int x [3];
	cout << "Enter a number:  ";
	cin >> x[0];
	cout << "Enter a number: ";
	cin >> x[1];
	cout << "Enter a number: ";
	cin >> x[2];
	sort(x, x+3);
	for (int i = 0; i < sizeof(x)/sizeof(x[0]); i++)
		cout << x[i] << '\n';
	return 0;
}

File: csc106/Assignment 3/sort3.java
// Oliver Tonnesen
// V00885732

import java.util.Scanner;
import java.util.Arrays;

public class sort3 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int[] x = new int[3];
		System.out.print("Enter a number: ");
		x[0] = sc.nextInt();
		System.out.print("Enter a number: ");
		x[1] = sc.nextInt();
		System.out.print("Enter a number: ");
		x[2] = sc.nextInt();
		Arrays.sort(x);
		for(int i = 0; i < x.length; i++) {
			System.out.println(x[i]); 
		}



	}

}
